WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -483,28 +486,44 @@ import WordPressFlux
 
     // MARK: - Setup
 
-    private func setupTableView() {
+    private func setupStackView() {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+
+        setupTableView(stackView: stackView)
+        setupJetpackBanner(stackView: stackView)
+
+        view.addSubview(stackView)
+        view.pinSubviewToAllEdges(stackView)
+    }
+
+    private func setupJetpackBanner(stackView: UIStackView) {
+        guard AppConfiguration.isWordPress && FeatureFlag.jetpackPowered.enabled else {
+            return
+        }
+
+        let jetpackBannerView = JetpackBannerView()
+        jetpackBannerView.setup()
```

Comment:
I think I should've set the `private` access modifier on that `setup()` method in `JetpackBannerView. 
This way we could simply call here `JetpackBannerView(frame: .zero)` and then maybe add `jetpackBannerView.translateAutoresizingMaskIntoConstraints = false`. What do you think?

Reply:
This should be resolved with 21479881509677ca1f1b5fc475cf523352e0b2b1. Thanks!